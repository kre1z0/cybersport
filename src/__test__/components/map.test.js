import React from 'react';
import ReactDOM from 'react-dom';
import { render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

import Map from '../../components/map';

const props = {
    center: [0, 0],
    resolution: 76.437,
};

describe('map component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Map {...props} />, div);
    });

    it('should mount map correctly', () => {
        const wrapper = mount(<Map {...props} />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it('should render map correctly', () => {
        const wrapper = render(<Map {...props} />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
