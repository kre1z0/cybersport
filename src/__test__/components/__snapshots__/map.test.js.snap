exports[`map component should mount map correctly 1`] = `
<Map
  center={
    Array [
      0,
      0,
    ]
  }
  onCenterChange={[Function]}
  onResolutionChange={[Function]}
  resolution={76.437}>
  <div
    className="map-container" />
</Map>
`;

exports[`map component should render map correctly 1`] = `
<div
  class="map-container" />
`;
